Chrysalis-Papilio/Soen-490-Papilio-Android
# Activity quiz

# General info

**Issue number**: PAPILIO-80

**Task description**: 

 - Connect activity quiz to the backend

# Testing
To test this feature you have to:

1. Login to the app
2. Click on the "Account" tab
3. Click on "Take the activity quiz"
4. Select indoor and/or outdoor activities and press "Next"
5. Select the activities of interest, press "Done"
6. Ensure you are redirected to the "Account" tab
7. Check the database to ensure this data was received
